### ISSUE TITLE
Proposal for a Sublineage of AY.23 with Spike 17del, L18I, T22I, P25L, 242/243del, E484Q, N501I, I1115V, V1264L (18 Sequences, Indonesia)

### ISSUE THREAD

**Issue by c19850727** [cov-lineages/pango-designation#393]
So there are 18 sequences uploaded on Jan. 2nd 2022 from Indonesia that seem like potential candidate of the second generation Delta. They are still too new to be visible on either Neherlab or Cov-spectrum, but I believe it's worth monitoring from now on.

**Description**
**Sub-lineage of**: AY.23
**Earliest sequence**: 2021/12/14
**Most recent sequence**: 2021/12/21
Countries circulating: Indonesia

Substitutions on top of AY.23:

<html>
<body>

Gene | Mutations
-- | --
ORF1a: | T170I, A1708D, A1716T
ORF1b: | H1948R
ORF3a: | T24I, P25S
ORF10: | R24C
Spike: | N17del, L18I, T22I, P25L, 242/243del, E484Q, N501I, I1115V
nuc | T4885C, C4921T, A10912G, C12295T, A13566G, C17550T, C24130T, C25096T

</body>
</html>

**Evidence**
Usher tree:
![image](https://user-images.githubusercontent.com/86835836/148004083-35f80ce4-c653-46b6-a450-3f338874ad20.png)
Mutations on branch as per Usher:
![image](https://user-images.githubusercontent.com/86835836/148004119-2b8fd16f-9d49-4865-9857-a8bf6948e711.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_c82a_3b2570.json?c=userOrOld&label=nuc%20mutations:C29627T

**Genomes**:
EPI_ISL_8273766, EPI_ISL_8294402-8294418

**Comment by c19850727** [comment]
So I just took another look at the metadata:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/itine/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/itine/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">



Accession | Collection   date | Location | Host   age | Host   gender | Host   status | Originating   lab | Submitting   lab
-- | -- | -- | -- | -- | -- | -- | --
EPI_ISL_8294418 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 25 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294417 | 2021/12/21 | Indonesia   / Banten / Tangerang | 22 | M | alive | KKP   Kelas 1 Soekarno-Hatta (Farmalab) | NIHRD
EPI_ISL_8294416 | 2021/12/19 | Indonesia   / Jakarta | 36 | M | alive | RSDC   Wisma Atlet Kemayoran | NIHRD
EPI_ISL_8294415 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 42 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294414 | 2021/12/14 | Indonesia   / Riau Islands / Batam | 35 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294413 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 40 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294412 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 30 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294411 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 39 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294410 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 33 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294409 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 28 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294408 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 44 | F | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294407 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 27 | F | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294406 | 2021/12/15 | Indonesia   / Riau Islands / Batam | 28 | M | alive | BTKLPP   Kelas I Batam | NIHRD
EPI_ISL_8294405 | 2021/12/17 | Indonesia   / Jakarta | 38 | F | alive | GSI   Laboratorium | NIHRD
EPI_ISL_8294404 | 2021/12/17 | Indonesia   / Jakarta | 34 | F | alive | GSI   Laboratorium | NIHRD
EPI_ISL_8294403 | 2021/12/17 | Indonesia   / Jakarta | 34 | F | alive | GSI   Laboratorium | NIHRD
EPI_ISL_8294402 | 2021/12/19 | Indonesia   / Jakarta | 27 | F | alive | RSDC   Wisma Atlet Kemayoran | NIHRD
EPI_ISL_8273766 | 2021/12/19 | Indonesia   / South Kalimantan | 27 | M | alive | RSUD   dr. H Moch. Ansari Saleh | NIHRD



</body>

</html>
Basically these 18 samples were originated from 5 different labs in 4 different regions across the archipelago, on 4 different days.

And Batam, Jakarta/Tangerang and South Kalimantan are 1k+ kms away from each other in different islands:
![image](https://user-images.githubusercontent.com/86835836/148009876-0f0edb63-abb6-4d74-89fd-eb3209058321.png)

**Comment by silcn** [comment]
Surely this is likely to have been an administrative error? 18 totally identical genomes suggests an extremely recent lineage (possibly only first-generation cases) and yet they're all over Indonesia. Even the initial B.1.1.529 samples had a small amount of diversity. Unless this is targeted sequencing of a known superspreader event, it seems more probable that there's been a mix-up, and instead of 18 different sequences, the same sequence was accidentally uploaded 18 times.

**Comment by c19850727** [comment]
Yes @silcn it looks suspicious to me as well, but there're actually some very minor differences in terms of NNNs. For example EPI_ISL_8294418 has Ns from 23350 to 23556; EPI_ISL_8294417 has Ns from 23350 to 23555, while others have Ns from 23350 to 23551.

**Comment by c19850727** [comment]
Just found yet another subclade of Delta from Indonesia with a completely unseen set of mutations, but again, these 12 sequences were all uploaded on the same day and identical to one another.

![image](https://user-images.githubusercontent.com/86835836/148623671-dad79fd5-6257-4d8d-a109-92830c49f9da.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_3a9a7_86e710.json?label=nuc%20mutations:G1890T,T13019C,T13620C,C21815A,T21831A,T21833G,T21837C,T21838G,T21842A,T21847C,C22388T,G25106C

Thus it seems more likely that all those subclades, as @silcn pointed out, are artefactual.

I'll close this proposal for now (but of course will continue keeping an eye on what's happening there)
